service-example--database/client/model_definition.yaml

```diff
@@ -0,0 +1,25 @@
+kind: Model
+api_version: v1beta1
+metadata:
+  display_name: Simple Client Access
+  name: client-service-access
+  type: model
+  publisher: DAFNI Example Models
+  summary: Write the ping output to a file
+  description: >
+    An example service type Model designed for use with DAFNI. Writes the ping output to a file caller ping.txt.
+    The IP address is passed in as a parameter.
+spec:
+  inputs:
+    parameters:
+      - name: INFLUX_IP
```

Comment:
This isn't really discussed - maybe it's part of the Workflow and therefore maybe documented elsewhere, but I don't understand how the user would ever know what this will be - or does it get set automatically?